keyboards/keebio/iris/rev2/config.h

```diff
@@ -19,10 +19,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xCB10
-#define PRODUCT_ID      0x1256
+#define PRODUCT_ID      0x2256
 #define DEVICE_VER      0x0200
 #define MANUFACTURER    Keebio
-#define PRODUCT         Iris Keyboard
+#define PRODUCT         Keebio Iris Rev. 2
```

Comment:
I'd like it on record that I absolutely hate and am vehemently opposed to repeating the Manufacturer name inside the Product string.

Reply:
I agree with @noroadsleft, it looks absolutely stupid on anything but Windows.

Reply:
Oh I know, just being a VIA shill I guess: https://caniusevia.com/docs/configuring_qmk#change-product